Vagrantfile

```diff
@@ -59,5 +59,7 @@ Vagrant.configure("2") do |box|
     # forward leap_web ports
     config.vm.network "forwarded_port", guest: 80,  host:8080
     config.vm.network "forwarded_port", guest: 443, host:4443
+    # forward pixelated ports
```

Comment:
oh, nevermind i oversaw your comment.

yes, please remove the existing 8080:80 mapping.